usl_pipeline/usl_lib/usl_lib/storage/metastore.py

```diff
@@ -0,0 +1,80 @@
+import dataclasses
+from typing import Optional
+
+from google.cloud import firestore
+
+
+STUDY_AREAS = "study_areas"
+CHUNKS = "chunks"
+
+
+@dataclasses.dataclass(slots=True)
+class StudyArea:
+    """A geography for either training or prediction.
+
+    Attributes:
+        name: A human-readable name of the study area. Must be unique.
+        col_count: Number of columns in the raster of the area.
+        row_count: Number of rows in the raster of the area.
+        x_ll_corner: X-coordinate of the raster's origin.
+        y_ll_corner: Y-coordinate of the raster's origin.
+        cell_size: Size of cells in the raster.
+        crs: Coordinate system connecting global geographic coordinates and local
+            projected ones.
+    """
+
+    name: str
+    col_count: int
+    row_count: int
+    x_ll_corner: float
+    y_ll_corner: float
+    cell_size: float
+    crs: str
+
+    def create(self, db: firestore.Client) -> None:
```

Comment:
I think metastore should also let caller load (look-up) StudyArea object by name.

Reply:
It will!
I was only doing the writing in this PR.

Reply:
I see. SG!